### ISSUE TITLE
Accessibility Issue - Inappropriate use of aria-pressed attribute

### ISSUE THREAD

**Issue by Bradrajkumar** [microsoft/BotFramework-WebChat#4469]
**Issue Description : -**

Interactive elements need a programmatically determined name, role, and value. These properties must also be accurate and appropriate.
There are button elements where state have been used unnecessarily. This may particularly disadvantage users that interface with the accessibility properties of elements, such as screen reader and voice recognition users.

**Success Criteria : -**

Remove aria-pressed attribute from the source code of the buttons. For example:

<button>JUST enough</button>
For more information, see WAI-ARIA Practices - Toggle Button. As well, for guidance on reviewing accessibility properties, see Chrome - Accessibility properties, Edge - Accessibility Pane, and Firefox - Accessibility Inspector.

![20_1](https://user-images.githubusercontent.com/34365788/197951878-0146725f-9b7c-4169-a400-41eceb75e817.png)
![20_2](https://user-images.githubusercontent.com/34365788/197951868-2f59dcef-9b74-4484-8831-09bb8a673bd4.png)

**Steps to reproduce**

1. Observe the interactive elements.
2. Inspect the elements and review the DOM.
3. Review the computed accessibility properties.
4. Observe that the aria-pressed is incorrectly used.

**Webchat Version :- 4.13.0**

**Comment by compulim** [comment]
> To DRI: Need more discussions with customers.

The buttons in the Adaptive Card are for making decisions. When the "JUST enough" button is clicked, a decision is being made and it will be highlighted.

When the end-user move the cursor and read the message again. The `aria-pressed="true"` will help them to identify which decision they had made, or in other words, which button they had clicked.

This is a screenshot from our sample:

![image](https://user-images.githubusercontent.com/1622400/198734911-d8f8b1bf-47be-4aa2-b40e-a13a9f7126df.png)

When the button is being clicked, the button will be highlighted (in blue).

We have some early accessibility reports saying the decision (i.e. button) cannot properly identified after the button is clicked. Thus, we added `aria-pressed` to make them a toggle button.

I read a few WAI-ARIA articles but I could not find anything that would say the current usage is not great. There are things that I read along with excerpts:

- [WAI-ARIA `aria-pressed` state](https://w3c.github.io/aria/#aria-pressed)
   - Toggle buttons require a full press-and-release cycle to change their value. Activating it once changes the value to true, and activating it another time changes the value back to false. A value of mixed means that the values of more than one item controlled by the button do not all share the same value. If the [attribute](https://dom.spec.whatwg.org/#concept-attribute) is not present, the button is not a toggle button.
- [WAI-ARIA-APG Button](https://www.w3.org/WAI/ARIA/apg/patterns/button/)
   - Toggle button: A two-state button that can be either off (not pressed) or on (pressed). To tell assistive technologies that a button is a toggle button, specify a value for the attribute [aria-pressed](https://w3c.github.io/aria/#aria-pressed). For example, a button labelled mute in an audio player could indicate that sound is muted by setting the pressed state true.

While I understand the button is not exactly toggleable: once it's pushed, it's pushed and cannot be popped. The `aria-pressed` seems still valid to use.

Could you:

- Tell us which part of WAI-ARIA, WAI-ARIA-APG or other WCAG articles we should read?
- Instead of removing `aria-pressed`, could you tell us if there are better alternatives that we could use to indicate a decision?

**Comment by compulim** [comment]
Pending for suggestions on: "what ARIA role/attributes to use for one-off decision making buttons".
